cmd/up.go

```diff
@@ -242,6 +242,7 @@ func ClusterUp(ctx context.Context, dialersOptions hosts.DialersOptions, flags c
 		return APIURL, caCrt, clientCert, clientKey, nil, err
 	}
 
+	data["calico-upgrade-cluster"] = reconcileCluster
```

Comment:
Could you elaborate a bit on what this mean? I saw we are checking it somewhere else

Reply:
`reconcileCluster` is a boolean which is set to true when rke is upgrading an existing cluster, false when creating a new one. I added the value to `data` as it was the simplest way to access it in `deployCalico`.